main/pipeline/pipeline.h

```diff
@@ -24,6 +25,7 @@ std::vector<core::FileRef> reserveFiles(std::unique_ptr<core::GlobalState> &gs,
 std::vector<ast::ParsedFile> index(core::GlobalState &gs, absl::Span<core::FileRef> files, const options::Options &opts,
                                    WorkerPool &workers, const std::unique_ptr<const OwnedKeyValueStore> &kvstore);
 
+std::unique_ptr<parser::Node> runPrismParser(core::GlobalState &gs, core::FileRef file);
```

Comment:
Added to the public API for use in tests. I will eventually move this into its own namespace so please don't worry about it for now 😁